src/Event/EventBundle/Entity/Media.php

```diff
@@ -0,0 +1,269 @@
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(name="copyrightInfo", type="text", nullable=true)
+     */
+    private $copyrightInfo;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(name="mediaCredits", type="string", nullable=true)
+     */
+    private $mediaCredits;
+
+    /**
+     * @var \DateTime
+     *
+     * @ORM\Column(name="created_date", type="datetime")
+     */
+    private $createdDate;
+
+    /**
+     * @var \DateTime
+     *
+     * @ORM\Column(name="updated_date", type="datetime")
+     */
+    private $updatedDate;
+    
+    public function getUploadImg(){
+        return '/uploads/media/' . $this->filename;
+    }
+
+    public function __construct()
+    {
+
+    }
+
+    /**
+     * Get id
+     *
+     * @return integer
+     */
+    public function getId()
+    {
+        return $this->id;
+    }
+
+    /**
+     * Set title
+     *
+     * @param string $title
+     * @return Media
+     */
+    public function setTitle($title)
+    {
+        $this->title = $title;
+
+        return $this;
+    }
+
+    /**
+     * Get title
+     *
+     * @return string
+     */
+    public function getTitle()
+    {
+        return $this->title;
+    }
+
+    /**
+     * Set filename
+     *
+     * @param string $filename
+     * @return Media
+     */
+    public function setFilename($filename)
+    {
+        $this->filename = $filename;
+
+        return $this;
+    }
+
+    /**
+     * Get filename
+     *
+     * @return string
+     */
+    public function getFilename()
+    {
+        return $this->filename;
+    }
+
+    /**
+     * Set description
+     *
+     * @param string $description
+     * @return Media
+     */
+    public function setDescription($description)
+    {
+        $this->description = $description;
+
+        return $this;
+    }
+
+    /**
+     * Get description
+     *
+     * @return string
+     */
+    public function getDescription()
+    {
+        return $this->description;
+    }
+
+    /**
+     * Set copyrightInfo
+     *
+     * @param string $copyrightInfo
+     * @return Media
+     */
+    public function setCopyrightInfo($copyrightInfo)
+    {
+        $this->copyrightInfo = $copyrightInfo;
+
+        return $this;
+    }
+
+    /**
+     * Get copyrightInfo
+     *
+     * @return string
+     */
+    public function getCopyrightInfo()
+    {
+        return $this->copyrightInfo;
+    }
+
+    /**
+     * Set mediaCredits
+     *
+     * @param string $mediaCredits
+     * @return Media
+     */
+    public function setMediaCredits($mediaCredits)
+    {
+        $this->mediaCredits = $mediaCredits;
+
+        return $this;
+    }
+
+    /**
+     * Get mediaCredits
+     *
+     * @return string
+     */
+    public function getMediaCredits()
+    {
+        return $this->mediaCredits;
+    }
+
+    /**
+     * Set createdDate
+     *
+     * @param \DateTime $createdDate
+     * @return Media
+     */
+    public function setCreatedDate($createdDate)
```

Comment:
add typehint

Reply:
I would replace `setCreated` and `setUpdated` with [Gedmo Timestampable](https://github.com/Atlantic18/DoctrineExtensions/blob/master/doc/timestampable.md#timestampable-document-example)